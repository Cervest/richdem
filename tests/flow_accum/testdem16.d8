ncols         15
nrows         15
xllcorner     421568
yllcorner     4872699
cellsize      3
NODATA_value  9
1 1 1 1 8 8 8 8 8 0 0 0 0 0 0
1 1 1 1 8 8 8 8 8 0 0 0 0 0 0
1 1 1 1 8 8 8 8 8 0 0 0 0 0 0
1 1 1 1 8 8 8 8 8 0 0 0 0 0 0
1 1 1 1 8 8 8 8 8 0 0 0 0 0 0
1 1 1 1 1 1 1 1 1 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0