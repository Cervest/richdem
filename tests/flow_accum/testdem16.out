ncols         15
nrows         15
xllcorner     421568
yllcorner     4872699
cellsize      3
NODATA_value  -1
   4   3   2   1   1   1   1   1   1   1   1   1   1   1   1
   5   4   3   2   2   2   2   2   1   1   1   1   1   1   1
   6   5   4   3   3   3   3   2   1   1   1   1   1   1   1
   7   6   5   4   4   4   3   2   1   1   1   1   1   1   1
   8   7   6   5   5   4   3   2   1   1   1   1   1   1   1
  24  23  22  21  15  10   6   3   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
   1   1   1   1   1   1   1   1   1   1   1   1   1   1   1
